app/admin/users.rb

```diff
@@ -0,0 +1,3 @@
+ActiveAdmin.register User do
+  permit_params :email, :password, :password_confirmation
```

Comment:
他の設定も管理者画面と同じでよいでしょう。

```suggestion
  permit_params :email, :password, :password_confirmation

  index do
    selectable_column
    id_column
    column :email
    column :current_sign_in_at
    column :sign_in_count
    column :created_at
    actions
  end

  filter :email
  filter :current_sign_in_at
  filter :sign_in_count
  filter :created_at

  form do |f|
    f.inputs do
      f.input :email
      f.input :password
      f.input :password_confirmation
    end
    f.actions
  end
```